src/frameworks/nextjs.ts

```
@@ -1,112 +1,358 @@
-  // Clear cookie using Next.js cookies() function
-  const cookieStore = cookies();
-  cookieStore.set(authConfig.cookieName, "", result.cookieOptions);
-  
-  return { message: result.message };
+  try {
+    const result = logoutCore();
+    
+    // Try to import and use cookies() dynamically
+    try {
+      // Use eval to avoid TypeScript compilation errors
+      const nextHeaders = eval('require("next/headers")');
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)